instrumentation/koala/test/instrumentation/koala/instrumentation_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+require_relative '../../../lib/opentelemetry/instrumentation/koala'
+
+describe OpenTelemetry::Instrumentation::Koala do
+  let(:instrumentation) { OpenTelemetry::Instrumentation::Koala::Instrumentation.instance }
+
+  it 'has #name' do
+    _(instrumentation.name).must_equal 'OpenTelemetry::Instrumentation::Koala'
+  end
+
+  it 'has #version' do
+    _(instrumentation.version).wont_be_nil
+    _(instrumentation.version).wont_be_empty
+  end
+
+  describe '#install' do
+    it 'accepts argument' do
+      instrumentation.install({})
+    end
+  end
+
+  describe 'present' do
+    it 'when koala gem installed' do
+      _(instrumentation.present?).must_equal true
+    end
+
+    # it 'when koala gem not installed' do
+    # hide_const('Koala')
+    # _(instrumentation.present?).must_equal false
```

Comment:
This tends to be a better test when you have a bit more doing on in the `present` block, this is effectively just testing that `defined?` works, not sure it's needed here.